adapters/InMobi/InMobiAdapter/GADMAdapterInMobiInterstitialAd.m

```diff
@@ -102,7 +102,12 @@ - (void)requestInterstitialAd {
   if (extras && extras.keywords) {
     [_interstitialAd setKeywords:extras.keywords];
   }
-
+  
```

Comment:
Nit: Our internal code review system detects empty spaces in this line, can you remove them?